### ISSUE TITLE
isPlatformPaySupported with existingPaymentMethodRequired false positive on Android

### ISSUE THREAD

**Issue by marchingband** [stripe/stripe-react-native#1371]
I have an Android device and simulator running an app where I have
```
    isPlatformPaySupported({
            googlePay: {
                existingPaymentMethodRequired: true, // must have card in wallet
            } 
        }).then(res=>{
            console.log("platformPaySupported: " + res)
            this.setState({platformPaySupported: res})
        })
```

I have Google Pay installed on the real device, with no cards in the wallet, I am in Canada so it's not even possible to add a card, and on the simulator, I have no google account setup at all, no apps, so certainly no cards in the wallet. `isPlatformPaySupported` return `true` anyway.
I would definitely expect it to return `false`

**Comment by charliecruzan-stripe** [comment]
@marchingband I'm landing a couple fixes today, expectation is that we'll release a patch update early this week!

**Comment by charliecruzan-stripe** [comment]
@marchingband released in 0.27.1

**Comment by marchingband** [comment]
That's so great thank you!
I am on expo though, so I am pinned to 0.23.3 until expo SDK 49 comes? Is that accurate?
